externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/SchemaHelper.scala

```diff
@@ -39,6 +44,7 @@ object SchemaHelper {
     case _: DecimalType => TTypeId.DECIMAL_TYPE
     case DateType => TTypeId.DATE_TYPE
     case TimestampType => TTypeId.TIMESTAMP_TYPE
+    case ntz if ntz.getClass.getName.equals(TIMESTAMP_NTZ) => TTypeId.TIMESTAMP_TYPE
```

Comment:
why not use simple class name as intervals？

Reply:
I found intervals did not handle the getColumnSize ,and I'll fix it in another pr。 So I would replace the string with  constant variables ，but maybe simple class name is a good choice。

Reply:
> why not use simple class name as intervals？

I found intervals did not handle the getColumnSize ,and I'll fix it in another pr。 So I would replace the string with constant variables ，but maybe simple class name is a good choice。

Reply:
+1 for simple class name